src/backend/libs/efiling-facade-client/src/main/java/ca/bc/gov/open/jag/efilingfacadeclient/service/config/EfilingFacadeConfig.java

```diff
@@ -0,0 +1,26 @@
+package ca.bc.gov.open.jag.efilingfacadeclient.service.config;
+
+import ca.bc.gov.open.jag.efilingfacadeclient.service.EfilingFacadeService;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.oxm.jaxb.Jaxb2Marshaller;
+
+@Configuration
+public class EfilingFacadeConfig {
+
+    @Bean
+    public Jaxb2Marshaller marshaller(){
+        Jaxb2Marshaller jaxb2Marshaller = new Jaxb2Marshaller();
+        jaxb2Marshaller.setContextPath("ca.bc.gov.open.jag.ag.csows.filing");
+        return jaxb2Marshaller;
+    }
+
+    @Bean
+    public EfilingFacadeService eFilingFacadeClient(Jaxb2Marshaller jaxb2Marshaller) {
+        EfilingFacadeService eFilingFacadeService = new EfilingFacadeService();
+        eFilingFacadeService.setDefaultUri("http://wsgw.dev.jag.gov.bc.ca:8080/csowsJBOSS/FilingFacade");
```

Comment:
#### Question
should this url be an env var?

Reply:
Moved it  to env variable and added properties to config